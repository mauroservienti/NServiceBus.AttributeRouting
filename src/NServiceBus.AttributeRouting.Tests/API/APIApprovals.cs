using ApprovalTests;
using ApprovalTests.Reporters;
using NUnit.Framework;
using PublicApiGenerator;
using System.Runtime.CompilerServices;

namespace NServiceBus.AttributeRouting.Tests.API
{
    public class APIApprovals
    {
        [Test]
        [UseReporter(typeof(DiffReporter))]
        [MethodImpl(MethodImplOptions.NoInlining)]
        public void Approve_API()
        {
            var publicApi = typeof(AttributeRoutingFeature).Assembly.GeneratePublicApi();

            Approvals.Verify(publicApi);
        }
    }
}