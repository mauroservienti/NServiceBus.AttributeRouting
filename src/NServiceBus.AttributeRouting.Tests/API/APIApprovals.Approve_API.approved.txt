[assembly: System.Reflection.AssemblyMetadata("RepositoryUrl", "")]
[assembly: System.Runtime.Versioning.TargetFramework(".NETStandard,Version=v2.0", FrameworkDisplayName="")]
namespace NServiceBus.AttributeRouting
{
    public class AttributeRoutingFeature : NServiceBus.Features.Feature
    {
        public AttributeRoutingFeature() { }
        protected override void Setup(NServiceBus.Features.FeatureConfigurationContext context) { }
    }
}
namespace NServiceBus
{
    public static class RoutingSettingsExtensions
    {
        public static void UseAttributeRouting(this NServiceBus.EndpointConfiguration endpointConfiguration) { }
    }
}